streaming/database.js

```diff
@@ -49,7 +49,7 @@ export function configFromEnv(env, environment) {
     if (typeof parsedUrl.password === 'string') baseConfig.password = parsedUrl.password;
     if (typeof parsedUrl.host === 'string') baseConfig.host = parsedUrl.host;
     if (typeof parsedUrl.user === 'string') baseConfig.user = parsedUrl.user;
-    if (typeof parsedUrl.port === 'string') {
+    if (typeof parsedUrl.port === 'string' && parsedUrl.port) {
```

Comment:
Perhaps: 
```suggestion
    if (typeof parsedUrl.port === 'string' && parsedUrl.port.length > 0) {
```

just so we don't accidentally not validate zeros or something due to type casting.

Reply:
You are right, while `'0' == false` being true is an interesting Javascript quirk, `'0'` is still truthy like any other non-empty string. Reverted that change.